tests/parser/FieldPropsTests.cpp

```diff
@@ -463,3 +463,36 @@ BOOST_AUTO_TEST_CASE(LATE_GET_SATFUNC) {
 
 }
 
+BOOST_AUTO_TEST_CASE(GET_TEMP) {
+    std::string deck_string = R"(
+GRID
+
+PORO
+   200*0.15 /
+
+)";
+
+    EclipseGrid grid(10,10, 2);
+    Deck deck = Parser{}.parseString(deck_string);
+    std::vector<int> actnum(200, 1); actnum[0] = 0;
+    grid.resetACTNUM(actnum);
+    FieldPropsManager fpm(deck, grid, TableManager());
+
+    BOOST_CHECK(!fpm.has<double>("NTG"));
+    const auto& ntg = fpm.get_copy<double>("NTG");
+    BOOST_CHECK(!fpm.has<double>("NTG"));
+    BOOST_CHECK(ntg.size() == grid.getNumActive());
+
+
+    BOOST_CHECK(fpm.has<double>("PORO"));
+    const auto& poro = fpm.get_copy<double>("PORO");
+    BOOST_CHECK(fpm.has<double>("PORO"));
```

Comment:
Please add this check another time to check whether it is possible to call `fpm.get_copy<double>("PORO");` twice. Please also check the expected size.

Reply:
I'll add those tests in a followup.